src/tests/ie_test_utils/common_test_utils/postgres_link.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "test_common.hpp"
```

Comment:
Why do you need "test_common.hpp" here?

Reply:
Probably, it isn't necessary. I inherited some class previously from TestCommon. Thanks!

Reply:
removed